travel.html

```diff
@@ -14,12 +14,12 @@ <h3 id="location-dates-times">Location / Dates / Times</h3>
                 <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1520.320051551835!2d-74.65377999190959!3d40.35032908895552!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c3e6c2c6e8d9f9%3A0x997e4572848080d8!2sFriend+Center+for+Engineering+Education%2C+William+St%2C+Princeton%2C+NJ+08540!5e0!3m2!1sen!2sus!4v1486700359730" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
             </div>
 
-            <p>You should plan on arriving around 5-6pm on Friday, March 31th and leaving around 3pm on Sunday, April 2nd.</p>
+            <p>You should plan on arriving around 5-6pm on Friday, November 10th and leaving around 3pm on Sunday, November 12th.</p>
 
 
             <h3 id="charter-buses">Charter Buses</h3>
 
-            <p>We will be sending one bus to the University of Waterloo and one bus to the University of Toronto and McMaster University. Since demand is high, spots on these buses will be available on a first-come-first-served basis for students at or traveling from near those schools. Once at HackPrinceton, we will reimburse the cost of your ticket at our travel booth. After you purchase your ticket, the bus captain will reach out to you via email with further details within a couple of weeks.</p>
+            <p>We will be sending charter buses out to help accommodate travel to and from certain schools. Since demand is high, spots on these buses will be available on a first-come-first-served basis for students at or traveling from near those schools. Once at HackPrinceton, we will reimburse the cost of your ticket at our travel booth. After you purchase your ticket, the bus captain will reach out to you via email with further details within a couple of weeks.</p>
 
             
             <h3 id="routes-and-schedules">Routes and Schedules</h3>
```

Comment:
Probably should delete this section

Reply:
@dfan97 @elizabethtian

Reply:
We can leave line 22, delete Routes and Schedules since those will change.